Kudu.Services/Deployment/OneDeployHelper.cs

```diff
@@ -134,6 +135,36 @@ public static bool EnsureValidStaticPath(ref string path, out string error, out
             return true;
         }
 
+        public static string GetArtifactNameFromAppSetting(ArtifactType artifactType)
```

Comment:
This is a great candidate for Unit Tests.  I would suggest to start practice that.